docs/infrastructure.md

```diff
@@ -0,0 +1,14 @@
+# Infrastructure for the Upstream team
+
+The Upstream team owns the following infrastructure.
+
+## GCP project `radicle-upstream`
+
+* Owners are @rudolfs, @juliendonck and @geigerzaehler
```

Comment:
GitHub markdown doesn't link the @mentions in the rendered file.

Reply:
Yeah, I don’t think that’s important